preview/index.ts

```diff
@@ -61,7 +69,13 @@ const getEmailNewslettersFromLocalCsv = async (): Promise<
 	});
 
 	logFeedback(cellsInRows[0][0], unvalidatedNewsletters);
-	return unvalidatedNewsletters.filter(EmailNewsletterType.is);
+
+	const includeInData = config.includeCancelled
```

Comment:
just a suggestion but wondering if this would read a little easier?

const allNewsletters = unvalidatedNewsletters.filter(EmailNewsletterType.is || CancelledEmailNewsletterType.is)
const activeNewsletters = allNewsletters.filter(EmailNewsletterType.is)

return includeCancelled ? allNewsletters : activeNewsletters